scripts/setup-hc-releases.sh

```diff
@@ -55,7 +55,6 @@ sums_name="hc-releases_${version}_SHA256SUMS"
 echo "Installing release $version"
 
 # download
-/bin/rm -vf "$sums_name" "hc-releases_${version}_${pattern}" # remove in case they already exist from previous invocation
```

Comment:
This is still needed unless the `gh release download ...` commands below get `--clobber`.
(When this was written, self-hosted runners had an older version of `gh` that didn't support `--clobber` and jobs that used the action twice would fail to download the second time around.)